src/vs/platform/webview/common/resourceLoader.ts

```diff
@@ -15,6 +15,10 @@ import { REMOTE_HOST_SCHEME } from 'vs/platform/remote/common/remoteHosts';
 import { IRequestService } from 'vs/platform/request/common/request';
 import { getWebviewContentMimeType } from 'vs/platform/webview/common/mimeTypes';
 
+
+export const webviewPartitionId = 'webview';
```

Comment:
Is this a temporary partition on purpose, because is only to intercept requests in a isolated way and not for persisting data to disk ?

Reply:
Yes we use partition to make sure that our  `onBeforeRequest` handler only intercepts requests from webviews instead of being fired for all requests

I've also added a filter so that we only intercept localhost requests from webviews

Reply:
Just to note, if the partition is not persistent that is, if it doesn't start with `persist:webview`, then the storage for webviews will be temporary, cookies, web storage etc, network auth cache etc or any user data related to the browsing context will not be persisted to disk. Is that the intention here ?

Reply:
yes, right now webviews are supposed to be entirely ephemeral

Reply:
got it, thanks for confirming!